src/components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { View } from 'react-native';
+import { select, text as textKnob } from '@storybook/addon-knobs';
+import { Card, CardTypes } from './Card';
+import { TextWithTheme } from '../../common/HelperComponents/ThemeHelper.styles';
+
+export default {
+  component: Card,
+  parameters: {
+    componentSubtitle:
+      'Cards are surfaces that groups content and actions in the same space. Its construction must always create a visual hierarchy of the content and communicate its purposes clearly.',
+  },
+  title: 'Components|Card',
+};
+
+const cardTypes = {
+  base: 'base',
+};
+
+export const base = () => (
+  <View
+    style={{
+      alignContent: 'center',
+      justifyContent: 'center',
+      margin: 30,
+      width: 328,
+    }}>
+    <TextWithTheme>Base</TextWithTheme>
+    <Card>
+      <View style={{ minHeight: 216 }} />
+    </Card>
+  </View>
+);
+
+export const interactive = () => (
+  <View
+    style={{
+      alignContent: 'center',
+      justifyContent: 'center',
+      margin: 30,
+      width: 328,
+    }}>
+    <Card type={select('Types', cardTypes, 'base') as CardTypes}>
+      <View>
+        <TextWithTheme style={{ margin: 16, minHeight: 216 }}>
+          {textKnob(
+            'Content',
+            'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc scelerisque metus nibh.',
```

Comment:
Idealmente, esse texto poderia ser algo descritivo.

Exemplo:
`This is the textual content of a card. You can use the Storybook knobs to change the content and understand how the card behaves.`

Existem alguns artigos que falam sobre a importância de [evitar Lorem Ipsum nos designs](https://www.secretstache.com/blog/stop-using-lorem-ipsum/) e até mesmo [evitar Lorem Ipsum em aplicações de exemplo](https://www.genui.com/resources/data-in-storybook)

> Validate that component design handles content beyond lorem ipsum